### DISCUSSION TITLE
Standard Charge Percent Validation Error Without Payer Note

### DISCUSSION THREAD

**Discussion by nahcat** [CMSgov/hospital-price-transparency#126] [Q&A] [answered]
I have a scenario where the methodology is percent of total charged, with a standard_charge_percent but none of the other standard charges.  In this scenario, the validator throws an error unless I include a payer note.  The conditional requirements in the data dictionary seem to imply each charge can stand on it's own.  Is there a requirement I'm missing in the documentation that states a payer note must be included if standard charge percent is the only value provided?

**Comment by RidenourK** [comment, marked as answer]
@nahcat. Thank you for your question. 

Would you be willing to provide a screenshot of the error you are receiving and a sample of the MRF that is triggering this error to the CMS HPT mailbox at [PriceTransparencyHospitalCharges@cms.hhs.gov](mailto:PriceTransparencyHospitalCharges@cms.hhs.gov) so we may investigate this further?
